R/count_watershed_data.R

```diff
@@ -81,15 +82,16 @@ count_watershed_data <- function(data_dir,
   import_raster(paste0(data_dir, file_paths["nlud"])) -> economic_USA
 
   # read NID point file
-  dams::nid_cleaned -> nid_dataset
+  dams::nid_subset -> nid_dataset
 
   nid_dataset %>%
-    filter(!is.na(Longitude),
-           !is.na(Latitude)) %>%
+    filter(!is.na(longitude),
+           !is.na(latitude)) %>%
     as_tibble() ->
     nid_no_NA
 
-  nid_spatial <- SpatialPointsDataFrame(coords = nid_no_NA[,c(6,7)],
+  nid_spatial <- SpatialPointsDataFrame(coords = nid_no_NA %>%
+                                          select(longitude, latitude),
```

Comment:
columns 6 and 7 do not refer to lat and lon in the updated file. This is why we want to avoid specifying columns by number in general.